kodi_addon_checker/schema_validation.py

```diff
@@ -115,7 +116,7 @@ def _validate(xml, schemapath):
 
 
 def check_version(branch_name, schema_file):
-    all_branches = ['leia', 'krypton', 'jarvis', 'isengard', 'helix', 'gotham']
+    all_branches = ValidKodiVersions[::-1]
```

Comment:
Why `::-1`?

Reply:
`ValidKodiVersions[::-1]` is `ValidKodiVersions` in reveres order

Reply:
Thank you, only found :-1 when googling. Which said it would be the array minus the last item.